packages/cms-base/components/public/cms/element/CmsElementText.vue

```diff
@@ -25,120 +25,124 @@ const style = computed<CSSProperties>(() => ({
 
 const hasVerticalAlignment = computed(() => !!style.value.alignItems);
 
-const CmsTextRender = () => {
-  const { resolveUrl } = useUrlResolver();
+const CmsTextRender = defineComponent({
```

Comment:
I've could also left it as it was to not use `defineComponent` and return wrapped nodes in `h` function. 

```ts
return () => h("div", {}, renderHtml(rawHtml, config, h, context, resolveUrl));
```